src/app/courses-list/courses-list.component.ts

```diff
@@ -36,14 +43,16 @@ export class CoursesListComponent implements OnInit {
      */
     public onNotify(value: number): void {
         console.log("Delete movie with ID# ", value);
+        this.dialog.open(PopupComponent, {
```

Comment:
also, you have to add afterClosed().subscribe... to react on dialog closing

Reply:
done